scheduler/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM balenalib/raspberrypi3-alpine-python
```

Comment:
@chrisys can we change this to `balenalib/raspberrypi3-alpine` ? I don't see any specific need for python right? and the smaller we can make this base the better.

Reply:
Sure, it was python as I was playing with some scripts to do other stuff that didn't make the cut